subsys/bluetooth/mesh/access.c

```diff
@@ -377,7 +377,7 @@ static int comp_add_elem(struct net_buf_simple *buf, struct bt_mesh_elem *elem,
 
 	if (net_buf_simple_tailroom(buf) < ((elem_size - *offset) + BT_MESH_MIC_SHORT)) {
 		if (IS_ENABLED(CONFIG_BT_MESH_LARGE_COMP_DATA_SRV)) {
-			/* Mesh Profile 1.1 Section 4.4.1.2.2:
+			/* MshPRTv1.1: 4.4.1.2.2:
```

Comment:
Should there be a acronyme desription for MshPRTv1.1, MshPRFv1.0.1 and MshMBTv1.0, or linking to BT Sig with this. So the user knows what this squashed word means. There maybe someone that wonders what MshPRTv1.1, MshPRFv1.0.1 and MBTv1.0 stands for.

Reply:
It is a fair point. If the majority of the references was directly in the documentation or API description I would definetly want a less cryptic acronym. When they are inside the stack implementation however I'm not sure if it is necessary. For most people that are qualified to go poking around in the stack source files I would expect that they att least can make a qualified guess what these acronyms refer to. But if you have a good suggestion where to place the acrony description I'm open to it 👍